dppylib/__init__.py

```diff
@@ -132,13 +132,32 @@ def insert_data(db, file_info):
         # Import data
         data_blob = []
         import_collection = db[file_info['collection']]
+        # Check for min and max day in collection
+        min_day = float('inf')
+        max_day = -1
+        min_day_found = import_collection.find_one({'day': {'$exists': True}}, sort=[('day', 1)], projection= {'day': 1})
+        if min_day_found is not None:
```

Comment:
I believe these conditions can be abridged as:

`if min_day_found:`

You can do that for future Python work.

Reply:
I figured this would be a little more strict since technically `if min_day_found` would not work if min_day_found were 0 or `False`. I know that `find_one` returns `None` for no record found, so I can be strict with the condition.